### PR TITLE
Implement find_images_by_nvrs by using Pyxis GraphQL

### PR DESCRIPTION
**Pull request by ronnyhlim**

### CODE REVIEW

**Comment by ronnyhlim** [comment]
@qixiang @mulaievaRH PTAL!

**Comment by ronnyhlim** [comment]
@qixiang @mulaievaRH  PTAL

File: tests/test_pyxis_gql.py [hunk 1]
```diff
@@ -321,6 +321,97 @@ def test_pyxis_graphql_find_images_by_nvr():
...
+            "page": 0,
+            "page_size": 50,
+            "total": 2,
+        }
+    }
+
+    pyxis_gql = PyxisGQL(url="graphql.pyxis.local", cert=("/path/to/crt", "/path/to/key"))
+    flexmock(Client).should_receive("execute").and_return(copy.deepcopy(result))
+
+    nvrs = ["python-v3.6-201910221723", "bash-4.2.46-34.el7.x86_64"]
```
Comment (qixiang) [inline tests/test_pyxis_gql.py:408]: This doesn't make sense, the mocked result only contains NVRs of `["foobar-container-v0.13.0-12.1582340001"]`, you should call it with the reasonable NVRs.

Comment (ronnyhlim) [reply tests/test_pyxis_gql.py]: changed the unit test to use "foobar-container-v0.13.0-12.1582340001" as discussed

**Review by qixiang — APPROVED** [review]
LGTM

**Pull request merged**